src/Dev/TestAssetStore.php

```diff
@@ -160,36 +160,11 @@ public static function getLocalPath(AssetContainer $asset, $forceProtected = fal
         return $relative ? $fileID : $adapter->prefixPath($fileID);
     }
 
-    public function cleanFilename($filename)
-    {
-        return parent::cleanFilename($filename);
-    }
-
     public function getFileID($filename, $hash, $variant = null)
     {
         return parent::getFileID($filename, $hash, $variant);
     }
 
-    public function parseFileID($fileID)
-    {
-        return parent::parseFileID($fileID);
-    }
-
-    public function getOriginalFilename($fileID)
-    {
-        return parent::getOriginalFilename($fileID);
-    }
-
-    public function getFilesystemFor($fileID)
```

Comment:
These methods shouldn't be removed - FlysystemAssetStore::getFilesystemFor() is a protected method, while this is a public method

Reply:
Done